src/main/java/org/jitsi/xmpp/component/ComponentBase.java

```diff
@@ -56,6 +61,12 @@
     private final static String PING_THRESHOLD_PNAME = "PING_THRESHOLD";
 
     /**
+     * The name of the property which configures {@link #processingTimeLimit}.
+     */
+    private final static String PROCESSING_TIME_LIMIT_PNAME
```

Comment:
Fixed in https://github.com/jitsi/jicoco/pull/20/commits/6c1cff497a037eddb7584231c56affb93ae740ef